networking/hardware_networks/about-sriov.adoc

```diff
@@ -15,7 +15,7 @@ The SR-IOV network device driver for the device determines how the VF is exposed
 * `netdevice` driver: A regular kernel network device in the `netns` of the container
 * `vfio-pci` driver: A character device mounted in the container
 
-You can use SR-IOV network devices with additional networks on your {product-title} cluster for application that require high bandwidth or low latency.
+You can use SR-IOV network devices with additional networks on your {product-title} cluster installed on bare metal infrastructure for application that require high bandwidth or low latency.
```

Comment:
```suggestion
You can use SR-IOV network devices with additional networks on your {product-title} cluster installed on bare metal infrastructure for applications that require high bandwidth or low latency.
```

Reply:
Also, should this sentence mention RHOSP as well as bare metal like in the module?